### ISSUE TITLE
Intellisense does not show parameters as you type - only on mouseover

### ISSUE THREAD

**Issue by PersonThing** [UnwrittenFun/svelte-vscode#84]
In a regular .html file, inside a `<script>` tag, you get intellisense for the parameters as you invoke a function.  In .svelte files, I don't.

.html file:
![image](https://user-images.githubusercontent.com/900715/66957017-151d1c00-f02b-11e9-9a3f-c5345a12e3a1.png)

.svelte file:
![image](https://user-images.githubusercontent.com/900715/66957123-3d0c7f80-f02b-11e9-9508-c3017246835e.png)

They both show full parameter (and jsdoc) info on hover of the method name... but it's really useful to be able to see which parameter you're working on as you type, especially for imported modules.
![image](https://user-images.githubusercontent.com/900715/66957222-67f6d380-f02b-11e9-9a26-981c1fc12f4a.png)

**Comment by UnwrittenFun** [comment]
You're right, seems to be signature help [(language server docs for it)](https://microsoft.github.io/language-server-protocol/specifications/specification-3-14/#textDocument_signatureHelp).

Going to transfer this issue to the language server and mark it as an enhancement
